js/social-comments.js

```diff
@@ -1,23 +1,91 @@
-const socialComments = document.querySelector('.social__comments');
+const blockBigPicture = document.querySelector('.big-picture'); //модальное окно с фото и информацией о нем
+const socialCommentsCount = blockBigPicture.querySelector('.social__comment-count'); //сообщение о количестве комментариев
+const commentsCount = socialCommentsCount.querySelector('.comments-count'); //число комментариев к фото
+const printedCommets = socialCommentsCount.querySelector('.printed-comments'); //отображение в разметке числа напечатанных комментариев
```

Comment:
```suggestion
const printedComments = socialCommentsCount.querySelector('.printed-comments'); //отображение в разметке числа напечатанных комментариев
```

поставь проверку орфографии в ide (если не знаешь как - спроси меня)